doc/user_guide/tools.rst

```diff
@@ -240,13 +240,48 @@ The methods of this section are available to all the signals. In other chapters
 methods that are only available in specialized
 subclasses.
 
+Simple mathematical operations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. versionchaned:: 0.9
```

Comment:
Version added has no text and, therefore, it doesn't appear. What about "Passing multiple axes in a tuple."?